odoo/addons/base/models/res_users.py

```diff
@@ -421,6 +425,29 @@ def _compute_share(self):
     def _compute_companies_count(self):
         self.companies_count = self.env['res.company'].sudo().search_count([])
 
+    def _compute_initials(self):
```

Comment:
The `_compute_` prefix is usually for functions associated with a computed field. Here it could be a regular function, even just inlined.

Reply:
The other function is somewhat similar. It's just a normal function returning stuff.